components/navbar/navbar_info_button.jsx

```diff
@@ -36,28 +37,37 @@ export default class NavbarInfoButton extends React.PureComponent {
                     />
                 );
             } else {
-                const link = (
-                    <a
-                        href='#'
-                        onClick={this.showEditChannelHeaderModal}
-                    >
-                        <FormattedMessage
-                            id='navbar.click'
-                            defaultMessage='Click here'
-                        />
-                    </a>
-                );
+                let addOne;
+                if (!this.props.isReadOnly) {
+                    const link = (
+                        <a
+                            href='#'
+                            onClick={this.showEditChannelHeaderModal}
+                        >
+                            <FormattedMessage
+                                id='navbar.click'
+                                defaultMessage='Click here'
+                            />
+                        </a>
+                    );
+                    addOne = (
+                        <div>
+                            {link}{' '}
+                            <FormattedMessage
+                                id='navbar.addOne'
+                                defaultMessage='to add one.'
```

Comment:
Instead of concatenating the sentence, can you structure it like this so that the translator has more control over the sentence?
```javascript
addOne = (
    <div>
        <FormattedMessage
            id='navbar.clickToAddHeader'
            defaultMessage='{clickHere} to add one.'
            values={{
                clickHere: link
            }}
        />
    </div>
);
```